src/components/AppIcon/AppIcon.css

```diff
@@ -0,0 +1,78 @@
+/**
+ * AppIcon
+ */
+
+@import "@folio/stripes-components/lib/variables";
+
+/**
+ * Icon
+ */
+
+.appIcon {
+  display: inline-flex;
+  align-items: center;
+}
+
+/**
+ * Label
+ */
+
+.label {
+  margin: 0 0 0 0.35em;
+}
+
+:global([dir="rtl"]) .label {
```

Comment:
Probably need to copy https://github.com/folio-org/stripes-components/blob/master/.stylelintrc to this repo, or at least make an exception for `:global` in this repo as exists in `stripes-components` ... it's a weird instance for 'global' to be required, since it isn't a class name that it's wrapping - @rasmuswoelk any thoughts?

Reply:
I don't think it's needed actually. I think it can be removed.